test/android-specs.js

```diff
@@ -122,7 +122,7 @@ describe('android', function () {
       mocks.verify();
     });
     it('fix', async function () {
-      (await check.fix()).should.equal('bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it');
+      (await check.fix()).should.equal(`${'bundletool.jar'.bold} is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it`);
```

Comment:
I'd rather check for simple text inclusion here to not overcomplicate tests with color modifiers

Reply:
same below